components/storage/StorageServiceCard.tsx

```diff
@@ -24,6 +24,32 @@ const StorageServiceCard: React.FC<ServiceCardProps> = ({ service, isDisabled })
     return String(feature.value);
   };
 
+  // Helper to sort features according to question order, then alphabetically
+  const sortFeatures = (features: ServiceFeature[]): ServiceFeature[] => {
```

Comment:
Oohhh this one is sooo much easier to understand. Also I think you're defining this twice in this PR. Can there just be one definition that you use to import?